content/pages/download-va-letters.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
```

Comment:
Let's add `template: level2-index` to the metadata to make the formatting consistent.

Reply:
@cvalarida would adding `permalink` here make sense as well?

Reply:
The page title has changed to `Download VA Benefit Letters and Documents` but the header is supposed to stay as `Download Your VA Letters`. Is there a way to use `template: level2-index` without having it pull the page title into the header? Or should I leave it out?

Reply:
Good question, looping in @cvalarida, should we add a `template_title` variable to the metadata to supply this information, which could then be used by the `level2-index` template to replace the default `title`, if provided?
Something like:
```
<h1>{{ template_title or title }}</h1>
```

Reply:
Never mind, I've been informed the change applies to both the page title and the header :)